debian/control

```diff
@@ -3,15 +3,46 @@ Section: web
 Priority: optional
 Maintainer: Debian letsencrypt <letsencrypt-devel@lists.alioth.debian.org>
 Uploaders: Francois Marier <francois@debian.org>
-Build-Depends: debhelper (>= 9), python-all, dh-python, python-nose, python-nosexcover, python-tox, python-docutils, python-sphinx, python-sphinx-rtd-theme, python-repoze.sphinx.autointerface
+Build-Depends: debhelper (>= 9),
+               dh-python,
+               python-all,
+               python-configargparse,
+               python-dialog (>=3.2.2rc1),
+               python-docutils,
+               python-nose,
+               python-nosexcover,
```

Comment:
I'm hoping we can keep running the tests at build time (hence a build dependency only), but yes, I don't plan on shipping them onto the final systems.